src/popup/router/components/AccountCard.vue

```diff
@@ -1,45 +1,37 @@
 <template>
   <div
-    :class="['account-card', { subaccount: idx !== 0, 'first-subaccount': idx === 1 }]"
+    :class="[
+      'account-card',
+      { subaccount: idx !== 0, 'first-subaccount': idx === 1 }]"
     :style="cssVar"
   >
     <AccountInfo v-bind="$attrs" />
-    <BalanceInfo v-bind="$attrs" />
-    <div class="arrows">
```

Comment:
class="arrow" related css should be also removed
https://github.com/aeternity/superhero-wallet/blob/44939e695dd12c46738e1e5c1a99359bb4163849/src/popup/router/components/AccountCard.vue#L95-L118